packages/jest-each/src/__tests__/index.test.ts

```diff
@@ -30,4 +30,77 @@ describe('template', () => {
       expect(a + b).toBe(expected);
     });
   });
+
+  describe('comments', () => {
+    describe('removes commented out tests', () => {
+      let testCount = 0;
+      const expectedTestCount = 2;
+
+      each`
+      a    | b    | expected
+      // ${1} | ${1} | ${0}
```

Comment:
What about commenting out a title line? Not sure if there's a good use case but should definitely have a test case to specify how it behaves.

Reply:
Throw on missing headngs...possibly should be in a different PR: https://github.com/facebook/jest/pull/8717/commits/065155a7e9d35c588ac75f70e2efa455164001a3.

https://github.com/facebook/jest/pull/8717/commits/aff5aaaa1265b86dacbc5f2777e02693a2a1ab49#diff-a6c0398707b866880e5098b277508a6dR474